Binary files fbgrab.old/fbgrab and fbgrab/fbgrab differ
Binary files fbgrab.old/fbgrab.1.gz and fbgrab/fbgrab.1.gz differ
diff -rupN fbgrab.old/fbgrab.c fbgrab/fbgrab.c
--- fbgrab.old/fbgrab.c	2015-10-19 18:15:05.824282499 +0300
+++ fbgrab/fbgrab.c	2015-10-19 18:16:03.376283925 +0300
@@ -313,7 +313,7 @@ static void write_PNG(unsigned char *out
     
     png_init_io(png_ptr, outfile);
     
-    png_set_compression_level(png_ptr, Z_BEST_COMPRESSION);
+    png_set_compression_level(png_ptr, compression);
     
     bit_depth = 8;
     color_type = PNG_COLOR_TYPE_RGB_ALPHA;
